label_studio/core/old_ls_migration.py

```diff
@@ -42,15 +42,15 @@ def suppress_autotime(model, fields):
 def _migrate_tasks(project_path, project):
     """ Migrate tasks from json file to database objects"""
     tasks_path = project_path / 'tasks.json'
-    with io.open(os.path.abspath(tasks_path)) as t:
+    with io.open(os.path.abspath(tasks_path), encoding='utf-8') as t:
```

Comment:
do we have a github issue patched by this fix?

Reply:
I didn't see it, but old projects with non utf8 chars won't be migrated